### DISCUSSION TITLE
AI復活隱私權疑慮

### DISCUSSION THREAD

**Discussion by Eileen050824** [1101-GS4719-C/Python-Community-GS4719-C#1944] [General]
有名中國網友將已逝藝人李玟、高以翔等人「AI復活」，相關影片在社群平台上轉傳引起議論。隨著生成式AI技術發展，「AI復活」近期在中國成為一門生意，提供客製化影片製作。在我看來，這類技術應用雖然可以滿足情感需求，然而卻也面臨侵權、隱私權洩露等諸多隱患。特別是未經親屬同意，將已故名人藉由「AI復活」後用於營利等商業活動，除了構成侵權行為的法律問題外，也無疑是再次揭家屬的傷疤。
我認為此舉或許多少能撫慰生者，就像黑鏡第二季第一集中，女主因難以忍受男友死於車禍的痛苦，便透過合成的模仿聲音和男友交流，繼而使用AI克隆過世的男友。現在有了AI技術的加持，一切好像就沒那麼遙遠！通過AI和逝者相聚，不僅能得到一絲慰藉，還能在難過時和已故的親友傾訴。但我換個方向想，卻也覺得這個做法沒什麼意義，人沒了就是沒了，自欺欺人的方法只會讓自己陷入虛無，更難走出失去親人的痛苦。而且用逝者生前影響克隆出的親人，還是我們認識的那個人嗎？不知道大家怎麼看，或是是否支持？
資料來源：聯合新聞網

**Comment by wanka2030** [comment]
AI如果能將人類徹底復活的話，豬大哥的大尾鱸鰻3是不是就可以準備開拍了(X
如果需要將人類的思緒也一併喚醒的話，要做到的應該是能夠有著原本生前的語氣，亦或者能跟著與人之間的對話能藉間改變對話的資料庫。影片能做到的只有將人生前的影像進行復課或重製，而人類的情感早就消逝。
如同「水星記」中的機器人取代了因車禍死去的父親，雖可以帶給留在現世的親人有所寄託，但始終無法解決親人死去的問題。

**Comment by SusuTima** [comment]
我覺得有這個技術讓我們懷念逝去的人是很好的，但是人走了，活著的人就要想辦法走出來，如果一直欺騙自己他還活著，一直走不出來，感覺這個技術就並不會幫到人，反而會害人深陷其中。

**Comment by jeff110303549** [comment]
我覺得這個議題非常有討論的價值，首先我也非常認同版主對於侵權的想法，不過在這個資訊爆炸的時代，維護隱私權似乎變得非常困難；再來是關於這項AI復活的技術，我認為這項科技能夠很大限度的撫慰那些剛經歷失去所愛的人，不過在使用這項技術的同時更應該知道逝去的將不再回來，一定要隨時告訴自己振作起來並重新回到生活避免過度的沉迷其中。

**Comment by happy246810** [comment]
對於這項技術，我是抱持著支持的，但是只限於非營利使用，而不是把它當成賺錢的工具，因為一旦被用來作為賺錢的工具，就會產生許多問題，像賺的錢是屬於誰的，是使用技術的人，還是已故名人的家屬。

**Comment by YANRONG0123** [comment]
我曾經在倫理學與美好生活中看過類似的運用，至於舉的例子是哪一位死者，能夠引起全班議論，相信各位心裡都有個底，科技雖然可以帶給我們方便，但我認為很多時候也造成更大的不尊重，這樣的行為覺得是不道德的。

**Comment by lyykey** [comment]
在我的看法中，將已逝親人透過AI復活這個概念確實讓人思索良久。雖然這樣的技術應用或許能夠緩解生者的情感痛苦，但同時也引發了許多法律、倫理上的問題。尤其是在未經親屬同意的情況下，將已故名人用於商業活動，不僅可能觸犯侵權行為，也是對於親屬的一種再次傷害。

**Comment by lyykey** [comment]
主角為了應對失去男友的痛苦，使用AI克隆出了男友的模擬聲音，這讓我們對於AI技術的可塑性有了更深的思考。但在我看來，雖然透過AI與逝者相聚或許能夠帶來一絲慰藉，但這種方法也許只是一種自我欺騙，無法真正幫助我們走出失去親人的痛苦。而且使用AI克隆出的親人是否還是我們所認識的那個人，這也是一個值得思考的問題。

**Comment by lyykey** [comment]
我認為這樣的技術應用雖然有其意義，但同時也存在著許多倫理和道德上的爭議。我們需要更深入地討論和思考這些問題，並且在利用這些技術時要更加謹慎和尊重生者及其家屬的意願。

**Comment by willychiuchiu** [comment]
AI復活技術看似非常強大，但我認為其弊還是大於利的，即使可能在某些正當情況下需要用到，但把他拿來做不正當行為的肯定佔大多數，例如以死者的名義來進行謀利，或是用來詐騙等等，這在法律上也很難有一個明確的規範，所以真的還需要再仔細思考這所衍生的問題

**Comment by YuanWang08** [comment]
我是支持這項技術的，我認為科技就是要使生活更便利、更開心，如果AI技術可以達成這個目的，我覺得是可行的。但這項技術應該的使用我覺得政府要嚴格規範，不然會衍生出新世代的犯罪問題。

**Comment by yxin214** [comment]
我不支持AI復活，我認為一個人沒有被遺忘就不算死去，他活在我們內心的樣貌才是最真實的樣子，AI復活不過是以大眾希望的樣子繼續活著。不知道大家有沒有看過電影 人魚沉睡的家 ，電影中透過科技讓腦死的女主角在接收禮物後被動地作出微笑等動作，本來應是開心接受禮物的畫面卻讓我不寒而慄，我想這也是我不支持AI復活的理由之一。

**Comment by Hahahahbla** [comment]
這個話題確實非常複雜，涉及情感、倫理、法律等多方面的考量。AI復活已故名人的技術應用引起了廣泛的關注和討論。AI復活技術具有雙刃劍的特性，既能在情感上給予人們一定的安慰，也伴隨著法律、倫理和心理上的諸多挑戰。對於這類技術的應用，我們應該保持謹慎態度，尊重逝者和其家屬的意願，並在法律和倫理框架內進行規範。希望通過公開討論和理性思考，能夠找到一個合適的平衡點，使技術真正造福人類，而不是帶來更多的問題和困擾。

**Comment by jylin34** [comment]
我認為這樣的技術應用雖然有其意義，但同時也引發了許多倫理和道德上的爭議。我們需要更深入地討論和思考這些問題，並在利用這些技術時更加謹慎，尊重生者及其家屬的意願。

**Comment by titan92601** [comment]
我認為「AI復活」雖滿足情感需求，卻引發隱私、道德問題。未經同意商業使用已逝名人形象，構成侵權。技術可撫慰但應慎重，否則可能阻礙正確處理失去親人的痛苦。

**Comment by NcuJerryKao** [comment]
AI技術的發展無疑帶來了許多新的應用可能性，但同時也引發了不少道德和法律問題。中國網友將已逝藝人李玟、高以翔等人「AI復活」，在社群平台上廣泛流傳，這一現象就引發了廣泛的議論和關注。雖然這類技術應用能夠在一定程度上滿足人們的情感需求，提供一種與逝者“重聚”的方式，但它也帶來了侵權和隱私洩露的風險。尤其是當這些未經親屬同意的「AI復活」影片被用於營利或商業活動時，不僅涉及法律層面的侵權行為，還可能給逝者的家屬帶來二次傷害，揭開他們尚未癒合的傷疤。

這種技術的應用在某些情況下或許能夠撫慰生者的心靈，就像《黑鏡》第二季中女主角通過AI模仿聲音和克隆技術來“復活”過世男友，這種方式提供了一種虛擬的陪伴，讓人在悲傷中能有傾訴的對象。然而，從另一個角度來看，這種自欺欺人的方法或許並沒有真正的意義。失去親人是無法逆轉的事實，通過AI技術來復刻逝者，可能只會讓人陷入虛無，難以真正走出失去的痛苦。此外，這樣復刻出的“親人”真的還是我們所認識的那個人嗎？這其中的情感真實性和人際互動的價值都值得深思。

因此，在享受AI技術帶來便利和新奇的同時，我們也應該謹慎考慮其倫理和法律影響，確保技術的使用不會對他人造成傷害。與其依賴虛擬的重聚，不如更多關注現實中的人際關係和情感交流，這樣才能真正走出失去的陰影，找到內心的平靜與慰藉。

**Comment by alexender0104** [comment]
我認為AI系統依賴於大量數據來訓練和運行。這些數據通常來自用戶的行為、偏好、位置等，可能導致過度收集個人信息。這可能是未經同意使用，用戶的數據可能在未經明確同意的情況下被收集和使用，這違反了隱私權和數據保護法。

**Comment by JustAHedgehog** [comment]
其實死者重生技術已經商業化了，例如在2023年韓國科技公司DeepBrain AI早前就展示了一項名為「Re;memory」的技術，採用基於人工智能的虛擬人，來模擬逝者的外貌表情、聲音和動作，客戶只需支付1000英鎊，即可通過投射影像與已故親人「視像通話」。
[相關影片](https://www.youtube.com/watch?v=pZeXjCUs1ak)
至於為何有此產業的產生，在工程講座與專業倫理課程上老師有提出現代社會因為講求效率、標準化的因素，逼著現代人需要為守喪設個標準時間，然而很矛盾的是悲傷並不是對關係突然結束的反應，而是關係經歷變形並成為新事物的過程。既然是過程，那麼悲傷本就因人而異。如果生者不能夠透過喪禮、追思與紀念等儀式回到常軌，將可能會感到被停滯與被拋下的悲傷。因此我認為是這現象導致死者重生技術產業的誕生。

**Comment by Shang621** [comment]
人們擔心人工智能的發展會進一步侵犯隱私權。AI技術的應用使得個人數據更容易被收集、分析和濫用，引發了對隱私保護的疑慮。監管和政策制定對於保護個人隱私至關重要，需要建立更嚴格的數據保護法律和標準，並加強監管和執法力度。同時，個人也應提高數字素養，加強對個人數據的保護意識，避免隱私泄露和濫用。
